build/proxies-scenarios.yml

```diff
@@ -22,6 +22,8 @@ parameters:
     arguments: --scenario proxy-yarp        $(proxyJobs)  --property proxy=yarp --application.channel edge --application.framework net6.0
     supportsServerHttps: true
     supportsServerHttp2: true
+  - displayName: YARP-net50
+    arguments: --scenario proxy-yarp        $(proxyJobs)  --property proxy=yarp --application.channel edge --application.framework net5.0
```

Comment:
channel current in this case, or don't specify it (default)

Reply:
I would recommend to give it a different name, like yarp-net5.0 so that it appears automatically in the charts. Otherwise I will have to find a hacky way to distinguish the data coming from the two runs.

Reply:
Ah I see the `--property proxy=` now, thought it was going by the "Display name" first